main.go

```diff
@@ -130,7 +154,7 @@ func getTimeout() time.Duration {
 	taskTimeout := os.Getenv("TASK_TIMEOUT")
 	if taskTimeout == "" {
 		log.Println("Default timeout: 30s")
-		timeout, _ := time.ParseDuration("30s")
+		timeout, _ := time.ParseDuration("30m")
```

Comment:
I think also 30s is a good number. I temporarily changed it to 30m for debugging as 30s expiry was too short for debugging. Will bring it back to original.